### PR TITLE
Add FIC support to Static Route

### PR DESCRIPTION
**Pull request by shisoAqron**
Support FIC parameter(fic_gw_id) in Static Route.

### CODE REVIEW

File: ecl/network/v2/static_routes/testing/request_test.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 	th "github.com/nttcom/eclcloud/testhelper"
 )
 
-func TestList(t *testing.T) {
+func TestListStaticRoute(t *testing.T) {
```
Comment (ManabuMiwa) [inline ecl/network/v2/static_routes/testing/request_test.go]:
```suggestion
func TestListStaticRoutes(t *testing.T) {
```

Comment (shisoAqron) [reply ecl/network/v2/static_routes/testing/request_test.go]: Thank you. this suggestion is supported.

**Review by naoshark — APPROVED** [review]

**Review by ManabuMiwa — APPROVED** [review]
LGTM!

File: ecl/network/v2/static_routes/testing/request_test.go [hunk 2]
```diff
@@ -129,7 +130,7 @@ func TestUpdate(t *testing.T) {
 	th.AssertEquals(t, i.ID, "93aaec0f-1546-4062-88c5-93c397b93c03")
 }
 
-func TestDelete(t *testing.T) {
+func TestDeleteStaticRoutes(t *testing.T) {
```
Comment (shimisho) [inline ecl/network/v2/static_routes/testing/request_test.go]:
```suggestion
func TestDeleteStaticRoute(t *testing.T) {
```

**Review by shimisho — CHANGES_REQUESTED** [review]

**Review by shimisho — APPROVED** [review]
LGTM :)

**Pull request merged**